src/main/resources/db/changelog/changes/01-create-books-table.yml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: Mykola
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
MySQL has all primary keys not null by default, but you might as well leave it as it is for it to be more explicit, up to you